pkg/engine/template_generator.go

```diff
@@ -10,7 +10,6 @@ import (
 	"encoding/json"
 	"fmt"
 	"runtime/debug"
-	"strconv"
```

Comment:
did you run `make generate` on these changes?

Reply:
Yes, does something look off?

Reply:
I don't see a templates_generated.go diff

Reply:
What part of this changeset do we expect to see reflected in `templates_generated.go`?

Reply:
I guess none, I thought I saw a parts/ change earlier but it's actually all pkg/